src/domains/translations/translations.controller.ts

```diff
@@ -1,5 +1,112 @@
-import { Router } from "express";
-
+import { Router } from 'express';
+import TranslationsService from './translations.service';
+import { Controller } from '../../types/express';
+import prisma from '../../prismaClient';
 const router = Router();
 
-export default router;
\ No newline at end of file
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Translation:
+ *       type: object
+ *       properties:
+ *         id:
+ *           type: string
+ *         challengeId:
+ *           type: string
+ *         userId:
+ *           type: string
+ *         title:
+ *           type: string
+ *         content:
+ *           type: string
+ *         likeCount:
+ *           type: integer
+ *         createdAt:
+ *           type: string
+ *           format: date-time
+ */
+
+/**
+ * @swagger
+ * /api/challenges/{challengeId}/translations:
+ *   get:
+ *     summary: 특정 챌린지의 번역 목록 조회
+ *     tags:
+ *       - Translations
+ *     parameters:
+ *       - in: path
+ *         name: challengeId
+ *         required: true
+ *         description: 챌린지 ID
+ *         schema:
+ *           type: string
+ *       - in: query
+ *         name: page
+ *         required: false
+ *         description: 페이지 번호 (기본값 1)
+ *         schema:
+ *           type: integer
+ *     description: >
+ *       특정 챌린지에 제출된 번역 작업물 목록을 조회합니다.
+ *       작업물 목록 조회는 기획상 "챌린지 상세 페이지" 내에 "참여 현황" 영역에서 보여지며,
+ *       기본 추천수(likeCount) 순으로 정렬되어 조회됩니다.
+ *       또한 페이지네이션이 적용되며, `totalCount`(전체 작업물 개수)을 통해 페이지네이션 구현에 활용할 수 있습니다.
+ *     responses:
+ *       200:
+ *         description: 번역 목록 응답
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 success:
+ *                   type: boolean
+ *                 data:
+ *                   type: array
+ *                   items:
+ *                     $ref: '#/components/schemas/Translation'
+ *                 totalCount:
+ *                   type: integer
+ *       400:
+ *         description: 잘못된 요청
+ *       404:
+ *         description: 챌린지를 찾을 수 없음
+ */
+
+const getTranslations: Controller = async (req, res, next) => {
+  try {
+    const { challengeId } = req.params;
+    const page = parseInt(req.query.page as string) || 1;
+
+    if (!challengeId) {
+      return next({ statusCode: 400 });
+    }
+
+    const challenge = await prisma.challenge.findUnique({
+      where: { id: challengeId },
+    });
+
```

Comment:
Controller에서 바로 prisma를 사용하셨네요!

저희가 route -> Controller -> Service 레어이를 나눈 이유는 각 레이어가 담담하는 것을 구분 하기 위함이에요
이것을 위반하면 테스트나 재사용이 어려울 수 있어요 ㅎㅎ

Controller에서는 요청 응답에 대한 처리
Service는 데이터 베이스 로직에 대한 처리 만 담당하게 하면 좋아요

```suggestion

    const challenge = TranslationsService.findChallengeById(challengeId);
    
```

위와 같이 변경하고 prisma 관련 로직은 service에서 구현하는 것을 추천 드립니다!